doc/CONFIGURATION.md

```diff
@@ -896,66 +863,56 @@ const node = await createLibp2p({
 
 #### Customizing Transports
 
-Some Transports can be passed additional options when they are created. For example, `libp2p-webrtc-star` accepts an optional, custom `wrtc` implementation. In addition to libp2p passing itself and an `Upgrader` to handle connection upgrading, libp2p will also pass the options, if they are provided, from `config.transport`.
+Some Transports can be passed additional options when they are created. For example, [webRTC](../packages/transport-webrtc) accepts optional [DataChannel Options](https://github.com/libp2p/js-libp2p/blob/master/packages/transport-webrtc/src/stream.ts#L13-L17). In addition to libp2p passing itself and an `Upgrader` to handle connection upgrading, libp2p will also pass the options, if they are provided, from `config.transport`.
 
 ```js
 import { createLibp2p } from 'libp2p'
-import { webRTCStar } from '@libp2p/webrtc-star'
-import { mplex } from '@libp2p/mplex'
 import { yamux } from '@chainsafe/libp2p-yamux'
 import { noise } from '@chainsafe/libp2p-noise'
-import wrtc from 'wrtc'
+import { webRTC } from '@libp2p/webrtc'
 
-const webRTC = webRTCStar({
-  wrtc
-})
 
 const node = await createLibp2p({
   transports: [
-    webRTC.transport
-  ],
-  peerDiscovery: [
-    webRTC.discovery
+    webRTC({
+      dataChannel: {
+        maxMessageSize: 10
```

Comment:
Why is this config key necessary?  People will copy/paste this as good config so it's worth noting why we're overriding the default value here.  If the default value is wrong it should probably be changed instead.

Reply:
The purpose of this example was to hihglight that you can pass an optional `maxMessageSize` value as mentioned in the above paragraph.